src/sqlancer/questdb/test/QuestDBQueryPartitioningBase.java

```diff
@@ -28,7 +27,7 @@
         implements TestOracle<QuestDBGlobalState> {
 
     QuestDBSchema s;
-    QuestDBTables targetTables;
+    QuestDBTable targetTable; // single table
```

Comment:
This comment is not useful, since the type already specifies that this is a single table. The "why?" would be more interesting here. As this is a nitpick, merging the PR nevertheless.